CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 1.6.x (????-??-??)
+## Improvements
+- Compatibility with PIM community edition 1.6
+- Compatibility with Pim enterprise edition 1.6
+- Remove the customized product export as the PIM 1.6 supports same filtering features (completeness, last export date)
+
```

Comment:
Wh must also document the naming change of `Processor/Normalization/AttributeProcessor`